common/metrics/defs.go

```diff
@@ -865,8 +865,8 @@ const (
 	TransferStandbyQueueProcessorScope
 	// TransferActiveTaskActivityScope is the scope used for activity task processing by transfer queue processor
 	TransferActiveTaskActivityScope
-	// TransferActiveTaskDecisionScope is the scope used for workflow task processing by transfer queue processor
-	TransferActiveTaskDecisionScope
+	// TransferActiveTaskWorkflowTaskScope is the scope used for workflow task processing by transfer queue processor
+	TransferActiveTaskWorkflowTaskScope
```

Comment:
probably call it 'TransferActiveWorkflowTaskScope'

Reply:
I think those are different tasks. And actually there are bunch of other metrics which starts with `TransferActiveTask`. I would keep it as is for consistency.